zerver/lib/actions.py

```diff
@@ -2870,11 +2878,74 @@ def do_remove_default_stream(stream):
     DefaultStream.objects.filter(realm_id=realm_id, stream_id=stream_id).delete()
     notify_default_streams(realm_id)
 
+def do_create_default_stream_group(realm, group_name, streams):
+    # type: (Realm, Text, List[Stream]) -> None
+    default_streams = get_default_streams_for_realm(realm.id)
+    for stream in streams:
```

Comment:
We should probably add a function that restricts characters in the default stream group name.  I think probably reusing `check_stream_name` for this would be reasonable.